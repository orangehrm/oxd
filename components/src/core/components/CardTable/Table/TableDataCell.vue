<template>
  <div role="cell">
    <slot></slot>
  </div>
</template>

<script lang="ts">
import {defineComponent} from 'vue';

export default defineComponent({
  name: 'oxd-card-td',
});
</script>
