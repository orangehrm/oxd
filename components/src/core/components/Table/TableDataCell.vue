<template>
  <td>
    <slot></slot>
  </td>
</template>

<script lang="ts">
import {defineComponent} from 'vue';

export default defineComponent({
  name: 'oxd-td',
});
</script>
