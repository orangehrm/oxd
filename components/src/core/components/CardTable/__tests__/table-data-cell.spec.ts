import {mount} from '@vue/test-utils';
import TableDataCell from '@orangehrm/oxd/core/components/CardTable/Table/TableDataCell.vue';

describe('CardTable > TableDataCell.vue', () => {
  it('renders OXD CardTable > TableDataCell', () => {
    const wrapper = mount(TableDataCell, {
      props: {},
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
