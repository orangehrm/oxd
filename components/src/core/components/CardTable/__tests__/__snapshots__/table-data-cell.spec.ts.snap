// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTable > TableDataCell.vue renders OXD CardTable > TableDataCell 1`] = `<div class="oxd-table-cell" role="cell"></div>`;
